recipes/s2let/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.2.3":
+    url: "https://github.com/astro-informatics/s2let/archive/refs/tags/v2.2.3rc1.zip"
```

Comment:
This PR currently points to a release candidate. I will make a full release before this is merged.

Reply:
We prefer the tar.gz when you change it